### DISCUSSION TITLE
请问UE环境puerst大家使用读取excel数据表的方式是什么

### DISCUSSION THREAD

**Discussion by ELenDiL1529** [Tencent/puerts#1048] [Q&A]
如题 另外蓝图定义的datatable可以被puerts读取到吗？

**Comment by chexiongsheng** [comment]
datatable在issue有个讨论。
其次不用问XXX可不可以做到。puerts能调用蓝图api，带反射的原生api，纯c++ api，所以你在蓝图或者c++能干的事情，都可以调用它们来完成。

**Comment by lonelyasme** [comment]
直接写个ＥＸＥＣＬ生成工具，把ＥＸＥＣＬ生成成数据类就完事啦。

**Comment by chexiongsheng** [comment]
datatable可以读，搜索下issue
